packages/core/services/google-maps-api-wrapper.ts

```diff
@@ -1,65 +1,64 @@
 import { Injectable, NgZone } from '@angular/core';
 import { Observable, Observer } from 'rxjs';
 
-import * as mapTypes from './google-maps-types';
-import { Polyline, PolylineOptions } from './google-maps-types';
 import { MapsAPILoader } from './maps-api-loader/maps-api-loader';
 
-// todo: add types for this
-declare var google: any;
+interface IVisibility {
```

Comment:
This is an internal interface (not exported) and will probably be removed after #1725 since visibility is not a layer property but instead a **maps** property. I left this as is to not break callers.

If you would like me to change it I can, but I'm just expressing it will probably go away very soon.